docs/graphql/manual/auth/authentication/jwt.rst

```diff
@@ -130,6 +139,7 @@ JSON object:
      "jwk_url": "<optional-url-to-refresh-jwks>",
      "claims_namespace": "<optional-key-name-in-claims>",
      "claims_format": "json|stringified_json",
+     "claims_namespace_path":"<optional-json-path-to-the-claims>",
```

Comment:
why not put `claims_namespace_path` right below `claims_namespace` as they are so similar?

Reply:
I'll update it